test/jasmine/tests/validate_test.js

```diff
@@ -505,4 +505,25 @@ describe('Plotly.validate', function() {
             'In layout, key grid.subplots[2][0] is set to an invalid value (5)'
         );
     });
+
+    it('should detect opposite axis range slider attributes', function() {
+        var out = Plotly.validate([
+            {y: [1, 2]},
+            {y: [1, 2], yaxis: 'y2'},
+            {y: [1, 2], yaxis: 'y3'}
+        ], {
+            xaxis: {
+                rangeslider: {
+                    yaxis: { rangemode: 'auto' },
+                    yaxis2: { rangemode: 'fixed' },
+                    yaxis3: { range: [0, 1] }
+                }
+            },
+            yaxis: {},
+            yaxis2: {},
+            yaxis3: {}
+        });
+
+        expect(out).toBeUndefined();
```

Comment:
As promised in https://github.com/plotly/plotly.js/pull/2364/files#r171957082

Please note, the `_isSubplotObj` flag in the range slider opposite axis attributes is very important. Without it, this test would spit out:

![image](https://user-images.githubusercontent.com/6675409/37106500-fea22574-21ff-11e8-9b5f-6b9587697fe6.png)

that is `Plotly.validate` would think that `rangeslider.yaxis2` and `rangeslider.yaxis3` aren't part of the schema.

Reply:
Note that this appears to work in `plotly.py` as well.

Copying the new `plot-schema.json` over [there](https://github.com/plotly/plotly.py/blob/master/plotly/package_data/default-schema.json) gives:

![image](https://user-images.githubusercontent.com/6675409/37106742-96f1da5e-2200-11e8-8d49-e1101cb4322c.png)

i.e. does not fail like:

![image](https://user-images.githubusercontent.com/6675409/37106774-aa28c98e-2200-11e8-8eff-4b658660741a.png)

cc @cldougl @Kully